app/policies/project_policy.rb

```diff
@@ -21,13 +21,26 @@ def index?
     true
   end
 
+  def destroy?
+    admin? || record.maintainers.include?(user)
+  end
+
   class Scope < Scope
     def resolve
       if admin?
-        scope.all
+        scope.where(projects: { deactivated_at: nil })
```

Comment:
Ez csak egy suggestion, de mivel itt egyetlen táblán dolgozunk, ezért e helyett
```ruby
where(projects: { deactivated_at: nil })
```
tudjuk ezt is írni
```ruby
where(deactivated_at: nil)
```